Packs/Hoxhunt/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,217 @@
+#### Classifiers
+
+##### New: Hoxhunt-classifier
+
+-   New: Classifies Hoxhunt incidents
+    <~XSOAR> (Available from Cortex XSOAR 6.10.0).</~XSOAR>
```

Comment:
```suggestion
```
Sorry, I missed this in my previous comment here, I would do it by my self but I don't have permissions.
Please do it for all the RN entries.